### PR TITLE
Russian language added

### PR DESCRIPTION
**Pull request by constantinef**
I have made the translation into russian

### CODE REVIEW

File: src/store/config/index.ts [hunk 1]
```diff
@@ -19,7 +19,8 @@ export const defaultState = (): ConfigState => {
         { name: 'English', code: 'en' },
         { name: 'Français', code: 'fr' },
         { name: 'Chinese', code: 'cn' },
-        { name: 'Nederlands', code: 'nl' }
+        { name: 'Nederlands', code: 'nl' },
+        { name: 'Russian', code: 'ru' }
```
Comment (shinuza) [inline src/store/config/index.ts:23]: Could you please write replace `Russian` with `русский`?

**Review by shinuza — CHANGES_REQUESTED** [review]

**Comment by shinuza** [comment]
Hey, thanks for the contribution. There's just a slight change needed before we can merge this

File: src/store/config/index.ts (hunk 1 above)
Comment (constantinef) [reply src/store/config/index.ts]: Done =)

**Comment by shinuza** [comment]
Thank you sir 👍🏾

**Comment by cadriel** [comment]
Thanks for the submission!

**Pull request merged**